src/components/Todo/Todo.module.css

```diff
@@ -0,0 +1,5 @@
+.title {
+    font-family: 'Courier New', Courier, monospace;
+    text-align: center;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6103729)